docs/querying/sql-data-types.md

```diff
@@ -80,8 +82,42 @@ the `UNNEST` functionality available in some other SQL dialects. Refer to the do
 > they are handled in Druid SQL and in native queries. For example, expressions involving multi-value dimensions may be
 > incorrectly optimized by the Druid SQL planner: `multi_val_dim = 'a' AND multi_val_dim = 'b'` will be optimized to
 > `false`, even though it is possible for a single row to have both "a" and "b" as values for `multi_val_dim`. The
-> SQL behavior of multi-value dimensions will change in a future release to more closely align with their behavior
-> in native queries.
+> SQL behavior of multi-value dimensions may change in a future release to more closely align with their behavior
+> in native queries, but the [multi-value string functions](./sql-multivalue-string-functions.md) should be able to provide
+> nearly all possible native functionality.
+
+## Arrays
+Multi-value dimensions may also be converted to standard SQL arrays, either by explicitly converting them with `MV_TO_ARRAY`,
+or implicitly when used within the [array functions](./sql-array-functions.md). `ARRAY` types behave as standard SQL arrays, where
+grouping on them will group on the entire array of values instead of the implicit `UNNEST` that occurs when grouping on
```

Comment:
```suggestion
Grouping on `ARRAY` types results in grouping on the entire array of values instead of the implicit `UNNEST` that occurs when grouping on
```